src/frontend/packages/lib_classroom/src/components/DashboardClassroomForm/index.spec.tsx

```diff
@@ -16,18 +16,9 @@ jest.mock('lib-components', () => ({
     resource: {
       id: '1',
     },
-    static: {
-      img: {
-        bbbBackground: 'some_url',
```

Comment:
I guess we don't need this part in the standalone website, it is the static needed for classroom:
https://github.com/openfun/marsha/blob/master/src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/Update/ClassRoomUpdate.tsx#L17-L24

Reply:
Beware, they are still used here
![image](https://user-images.githubusercontent.com/720491/226874759-59ee6425-5ee3-4fb9-b7d8-c292b738f698.png)